tardis/plasma/properties/nlte.py

```diff
@@ -68,38 +66,38 @@ def calculate(self, level_boltzmann_factor, electron_densities,
         #He II ground state
         helium_population.ix[1,0] = 1.0
         #He III states
-        helium_population.ix[2,0] = self.calculate_helium_three(t_rad, w,
+        helium_population.ix[2,0] = HeliumNLTE.calculate_helium_three(t_rad, w,
             zeta_data, t_electrons, delta, g_electron, beta_rad,
-            ionization_data, electron_densities, g)
-        unnormalised = helium_population.sum()
-        normalised = helium_population.mul(number_density.ix[2] / unnormalised)
-        helium_population.update(normalised)
+            ionization_data, g)
+#        unnormalised = helium_population.sum()
+#        normalised = helium_population.mul(number_density.ix[2] /
+# unnormalised)
+#        helium_population.update(normalised)
         return helium_population
 
     @staticmethod
     def calculate_helium_one(g_electron, beta_rad, ionization_data,
-        level_boltzmann_factor, electron_densities, g, w):
+        level_boltzmann_factor, g, w):
         """
         Calculates the He I level population values, in equilibrium with the He II ground state.
         """
-        return level_boltzmann_factor.ix[2,0].mul(
-            g.ix[2,0], axis=0) * (1./(2*g.ix[2,1,0])) * \
-            (1/g_electron) * (1/(w**2)) * np.exp(
-            ionization_data.ionization_energy.ix[2,1] * beta_rad) * \
-            electron_densities
+        return level_boltzmann_factor.ix[2,0] * (1./(2*g.ix[2,1,0])) * \
+            (1/g_electron) * (1/(w**2.)) * np.exp(
+            ionization_data.ionization_energy.ix[2,1] * beta_rad)
 
     @staticmethod
     def calculate_helium_three(t_rad, w, zeta_data, t_electrons, delta,
-        g_electron, beta_rad, ionization_data, electron_densities, g):
+        g_electron, beta_rad, ionization_data, g):
         """
         Calculates the He III level population values.
         """
         zeta = PhiSahaNebular.get_zeta_values(zeta_data, 2, t_rad)[1]
-        he_three_population = (2 / electron_densities) * \
+        he_three_population = 2 * \
```

Comment:
To me `\` at the end of a line looks ugly and PEP recommends to use `( )` instead.
The added benefit of the 2nd approach is that most editors will indent the following lines differently which looks nicer, too.